config/printer-creality-ender3-v2-neo-2022.cfg

```diff
@@ -1,4 +1,4 @@
-# This file contains pin mappings for the stock 2020 Creality Ender 3
+# This file contains pin mappings for the stock 2022 Creality Ender 3
```

Comment:
maybe use 2020/2022? or was this introduced in 2022?

Reply:
Hi @thijstriemstra. This config is for the Creality Ender 3 V2 Neo . The Creality Ender 3 V2 Neo is the 2022 follow-up to the 2020 Creality Ender 3 V2. The configuration to the 2020 version can be found here: https://github.com/Klipper3d/klipper/blob/master/config/printer-creality-ender3-v2-2020.cfg

This is just fixing a typo I noticed in the printer config that had the year incorrect as it should read 2022 and not 2020.